dcat/rdf/dcat3.ttl

```diff
@@ -1,132 +1,152 @@
+  dcterms:description "DCATは、ウェブ上で公開されたデータ・カタログ間の相互運用性の促進を目的とするRDFの語彙です。このドキュメントでは、その利用のために、スキーマを定義し、例を提供します。データ・カタログ内のデータセットを記述するためにDCATを用いると、公開者が、発見可能性を増加させ、アプリケーションが複数のカタログのメタデータを容易に利用できるようになります。さらに、カタログの分散公開を可能にし、複数のサイトにまたがるデータセットの統合検索を促進します。集約されたDCATメタデータは、ディジタル保存を促進するためのマニフェスト・ファイルとして使用できます。"@ja ;
+  dcterms:description "Το DCAT είναι ένα RDF λεξιλόγιο που σχεδιάσθηκε για να κάνει εφικτή τη διαλειτουργικότητα μεταξύ καταλόγων δεδομένων στον Παγκόσμιο Ιστό. Χρησιμοποιώντας το DCAT για την περιγραφή συνόλων δεδομένων, οι εκδότες αυτών αυξάνουν την ανακαλυψιμότητα και επιτρέπουν στις εφαρμογές την εύκολη κατανάλωση μεταδεδομένων από πολλαπλούς καταλόγους. Επιπλέον, δίνει τη δυνατότητα για αποκεντρωμένη έκδοση και διάθεση καταλόγων και επιτρέπει δυνατότητες ενοποιημένης αναζήτησης μεταξύ διαφορετικών πηγών. Συγκεντρωτικά μεταδεδομένα που έχουν περιγραφεί με το DCAT μπορούν να χρησιμοποιηθούν σαν ένα δηλωτικό αρχείο (manifest file) ώστε να διευκολύνουν την ψηφιακή συντήρηση."@el ;
+  dcterms:description "هي أنطولوجية تسهل تبادل البيانات بين مختلف الفهارس على الوب. استخدام هذه الأنطولوجية يساعد على اكتشاف قوائم  البيانات المنشورة على الوب و يمكن التطبيقات المختلفة من الاستفادة أتوماتيكيا من البيانات المتاحة من مختلف الفهارس."@ar ;
+  dcterms:description "DCAT er et RDF-vokabular som har til formål at understøtte interoperabilitet mellem datakataloger udgivet på nettet. Ved at anvende DCAT til at beskrive datasæt i datakataloger, kan udgivere øge findbarhed og gøre det gøre det lettere for applikationer at anvende metadata fra forskellige kataloger. Derudover understøttes decentraliseret udstilling af kataloger og fødererede datasætsøgninger på tværs af websider. Aggregerede DCAT-metadata kan fungere som fortegnelsesfiler der kan understøtte digital bevaring. DCAT er defineret på http://www.w3.org/TR/vocab-dcat/. Enhver forskel mellem det normative dokument og dette schema er en fejl i dette schema."@da ;
+# TO BE ADDED BEFORE PUBLICATION IN W3C SPACE
+#  dcterms:issued ""^^xsd:date ;
   dcterms:license <https://creativecommons.org/licenses/by/4.0/> ;
-  dcterms:modified "2012-04-24"^^xsd:date ;
-  dcterms:modified "2013-09-20"^^xsd:date ;
-  dcterms:modified "2013-11-28"^^xsd:date ;
-  dcterms:modified "2017-12-19"^^xsd:date ;
-  dcterms:modified "2019" ;
   dcterms:modified "2020-11-30"^^xsd:date ;
   dcterms:modified "2021-03-09"^^xsd:date ;
   dcterms:modified "2021-04-08"^^xsd:date ;
   dcterms:modified "2021-06-23"^^xsd:date ;
   dcterms:modified "2021-09-27"^^xsd:date ;
   dcterms:modified "2022-03-22"^^xsd:date ;
-  dcterms:modified "2022-05-12"^^xsd:date ;
+  dcterms:modified "2022-05-23"^^xsd:date ;
+  dcterms:modified "2023-01-05"^^xsd:date ;
+  dcterms:publisher <https://www.w3.org/> ;
```

Comment:
This is conflating a web page and an organization. As we don't have, currently, a nice IRI identifying the organization, I think the easiest path we have is this:
```suggestion
  dcterms:publisher [ foaf:homepage <https://www.w3.org/> ] ;
```
This pattern is already used for people and their affiliation in this schema.
This could be enriched further, e.g.
```suggestion
  dcterms:publisher [
      a org:Organization;
      foaf:homepage <https://www.w3.org/>;
      foaf:name "World Wide Web Consortium (W3C)";
      rdfs:seeAlso <https://www.wikidata.org/entity/Q37033>;
  ] ;
```

Reply:
I have accepted the second option providing further erichment.